### PR TITLE
Added Pydio Demo and Link Cleanup

### LINKED ISSUE #2152: Section of (Game-)Hostingpanel
Hi,
i would like to see a section in this list of selfhosted Controllpanels for example: 
ISPConfig3 or for Games Pterodactyl.

I think this would be a great addition because it helps in Hosting multiple instances of websites or Games.
Maybe call it Hosting and Controll Panels. This would also be a place for YunoHost or HomelabOS. 

Cloudron could maybe also be added as it offers a great managed Docker Controll (2 Container are free).

### LINKED ISSUE #2697: Clarify that static sites generators are not listed here
https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2576#issuecomment-867140036

> Sorry, we don't list static site generators. Please submit it at https://staticsitegenerators.net/ and/or https://www.staticgen.com/

>>Thanks for the reply and the explanation. Can you update the guidelines accordingly? Currently, the non-listing of static site generators is not explained.

### LINKED ISSUE #2834: Add pydio demo
Pydio has a demo at https://pydio.com/en/demo.

### LINKED ISSUE #3090: Removal of Documentation Generator section?
All three entries have last commit dates of >2018

https://github.com/mil/markdown-tree
https://ricostacruz.com/flatdoc/
https://github.com/haldean/docstore

Happy to remove and fix toc

### PR DESCRIPTION
**Pull request by KieranRobson**
Closes #2834 
Closes #3090
Closes #2697 With  the addition of `**Static site generators are not listed within this list. ...`

Misc issues that could be closed
#2152  - Can be closed due to adition of `Games - Administrative Utilities & Control Panels`
#1592  - Can be closed with PR #2031
#1642 - Can be closed, adding software last commit 2017

------

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [ ] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [ ] Additions are inserted preserving alphabetical order.
- [ ] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [ ] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [ ] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [ ] Any software project you are adding to the list is actively maintained.
- [ ] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by nodiscc — COMMENTED** [review]
Reviewed and OK up to `RM Clerk - Last Commit 2019`, reminder to myself to review the remaining commits

**Review by nodiscc — COMMENTED** [review]
Reviewed up to `RM Compacted - Last Commit 2018`, reminder for myself to continue from there.

File: README.md [hunk 1]
```diff
@@ -806,7 +805,7 @@ A [news aggregator](https://en.wikipedia.org/wiki/News_aggregator), also termed
 - [Kriss Feed](https://tontof.net/kriss/feed/) - Simple and smart (or stupid) feed reader. ([Demo](https://tontof.net/feed/), [Source Code](https://github.com/tontof/kriss_feed)) `CC0-1.0` `PHP`
 - [Leed](https://github.com/LeedRSS/Leed) - Leed (for Light Feed) is a Free and minimalist RSS aggregator. `AGPL-3.0` `PHP`
 - [Leselys](https://github.com/toxinu/leselys) - Your very elegant RSS reader. `AGPL-3.0` `Python`
-- [Lite-Reader](https://github.com/cubny/lite-reader) - Read your feeds on your own machine with a simple and lite application. ([Demo](http://cubny.com/lite-reader/)) `BSD-3-Clause` `PHP`
+- [Lite-Reader](https://github.com/scubny/lite-reader) - Read your feeds on your own machine with a simple and lite application. ([Demo](http://cubny.com/lite-reader/)) `BSD-3-Clause` `PHP`
```
Comment (nodiscc) [inline README.md:808]: The old URL was correct, can you please revert this change?

File: README.md [hunk 2]
```diff
@@ -1404,7 +1404,6 @@ _Related: [Wikis](#wikis)_
 - [mkaczanowski pastebin](https://github.com/mkaczanowski/pastebin) - Simple, fast, feature-rich, standalone pastebin service. `MIT` `Rust`
 - [mojopaste](https://metacpan.org/dist/App-mojopaste) - Perl based pastebin. ([Demo](https://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `Artistic-2.0` `Perl`
 - [MokinToken](https://github.com/nexus-uw/mokintoken) - Clientside encrypted pastebin using tweetnacl. `Unlicense` `PHP`
-- [NoteHub](https://github.com/chmllr/NoteHub) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. `MIT` `Nodejs`
```
Comment (nodiscc) [inline README.md:1421]: Maybe NoteHub should not be removed? It looks like a very simple codebase which may not need much improvements/maintenance:

```
$ cloc NoteHub/
      21 text files.
      21 unique files.                              
       5 files ignored.

github.com/AlDanial/cloc v 1.86  T=0.30 s (56.1 files/s, 4980.4 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Go                               6            105              8            804
CSS                              1             52              1            229
HTML                             4              0              0            138
Markdown                         2             29              0             78
JavaScript                       2              5              0             44
TOML                             1              1              0              6
make                             1              2              0              6
-------------------------------------------------------------------------------
SUM:                            17            194              9           1305
-------------------------------------------------------------------------------
```

@chmllr do you still use/maintain NoteHub?

**Review by nodiscc — COMMENTED** [review]
Thank you for taking the time to work on this. Let's wait some time for feedback from @Finesse @chmllr @schollz regarding the status of web-fonts-repository/NoteHub/Cowyo

**Comment by KieranRobson** [comment]
Travelling currently, the scuby link looks to be a typo, will fix

I removed them based on their last commits, for something simple such as notes and wikis, they probably don't need to be removed, will add back once feedback has been given from the mentioned users!

Travelling currently so will be a couple of hours.

File: README.md (hunk 2 above)
Comment (chmllr) [reply README.md:1421]: No I don't.

**Comment by KieranRobson** [comment]
> > do you still use/maintain Cowyo?
> 
> Yes and yes. It needs very little maintenance now, its been running steadily and working well for the past five years. I plan to keep it up long-term.
> 

Awesome, I'll revert my change!

File: README.md [hunk 3]
```diff
@@ -1269,7 +1269,6 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [TeslaMate](https://github.com/adriankumpf/teslamate) - A powerful data logger for Tesla vehicles. `MIT` `Elixir`
 - [Trello Burndown](https://github.com/mtricht/trello-burndown) `⚠` - Easy to use SCRUM burndown chart for Trello boards. `MIT` `Go/Docker`
 - [ViMbAdmin](https://www.vimbadmin.net/) - Provides a web based virtual mailbox administration system to allow mail administrators to easily manage domains, mailboxes and aliases. ([Source Code](https://github.com/opensolutions/ViMbAdmin)) `GPL-3.0` `PHP`
-- [Web fonts repository](https://github.com/Finesse/web-fonts-repository) - Simple webfont hosting. Google Fonts alternative for your own fonts. `MIT` `PHP`
```
[the remark this answers is not in the document]
Comment (Finesse) [reply README.md:1279]: Yes
